index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css" /> 
+    <script src="https://kit.fontawesome.com/26c38390b4.js" crossorigin="anonymous"></script>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <div class="background"><img src="./headline-mobile.png"></div>
+    <div class="top">
+        <div class="header">
+            <p class="logo">My Logo</p>
+        </div>
+        <div class="menuhover">
+            <button>
+                 <i class="fa-solid fa-bars"></i>
+            </button>
+        </div>
+    </div>
+    <div class="intro">
+        <p class="text">Im Gerson Higgins. Glad to see you!</p>
```

Comment:
- [ ] [OPTIONAL] The main title (e.g., "Im Gerson Higgins. Glad to see you!") is an `h1` tag. * If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

**_I am marking this comment as optional because the previous reviewer did not mention it. But I strongly suggest you fix these issues as they are core requirements that the previous reviewer missed._**